### PR TITLE
initial app creation

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@Rizziepit ready for review. 
It's just a bare app with the same structure as `unicore.hub`. I haven't made any decisions on data store so I removed all alchemy stuff.

**Comment by Rizziepit** [comment]
+1.

Once you've fleshed out `unicore.ask` we should discuss what we need in terms of app authorization in `unicore.hub`, if anything. I'm not certain how it should be done yet.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+install:
+  - pip install -r requirements-dev.txt --use-wheel
+  - pip install -e . --use-wheel
```
Comment (smn) [inline .travis.yml]: I think `--use-wheel` is now the default for recent pips

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+script:
+  - find ./unicore/ask -name '*.py' -not -path '*/alembic/*' | xargs pep8
+  - find ./unicore/ask -name '*.py' -not -path '*/alembic/*' | xargs pyflakes
```
Comment (smn) [inline .travis.yml]: I've enjoyed using `flake8` for this instead lately, simpler.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          "Programming Language :: Python",
+          "Framework :: Pyramid",
+          "Topic :: Internet :: WWW/HTTP",
+          "Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
+      ],
+      author='Praekelt Foundation',
+      author_email='dev@praekelt.com',
+      url='https://github.com/universalcore/unicore.ask.git',
+      license='BSD',
+      keywords='authentication, universal, core',
```
Comment (smn) [inline setup.py]: `authentication`?

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      include_package_data=True,
+      zip_safe=False,
+      namespace_packages=['unicore'],
+      install_requires=requires,
+      tests_require=requires,
+      entry_points="""\
+      [paste.app_factory]
+      main = unicore.ask.service:main
+      [console_scripts]
+      askservice=unicore.ask.service.commands:in_app_env
```
Comment (smn) [inline setup.py]: What's this?

**Comment by smn** [comment]
Sorry some late post-merge questions from me.

**Comment by smn** [comment]
Not urgent at all.

**Comment by miltontony** [comment]
@smn thanks. I've fixed those in develop. Mostly artefacts from `unicore.hub`.  I use it as a skeleton for this app.

**Pull request merged**